amazon-s3/6.0/modules/ROOT/pages/amazon-s3-connector-examples.adoc

```diff
@@ -0,0 +1,153 @@
+= Amazon S3 Connector 6.0 Examples
+:page-aliases: connectors::amazon/amazon-s3-connector-examples.adoc
+
+The following example shows how to store an image from a URL in Amazon S3, and retrieve and display the image.
+
+== Before You Begin
+
+For this example to work in Anypoint Studio, you must provide Amazon Web Services credentials. You can either:
+
+* Replace the variables with their values in the code.
+* Provide the values for each variable in the `src/main/resources/mule-artifact.properties` file.
+
+[[store-and-retrieve]]
+== Example Steps
+
+image::amazon-s3-02.png["Studio 7 Visual Studio Icon Flow. The flow shows Listener, Create bucket, Get MuleSoft logo, Create logo object in S3 bucket, Get Image, and Delete S3 bucket."]
+
+. Create a new Mule project in Studio.
+. Drag an *HTTP > Listener* into the canvas, and select it to open the properties editor.
+. Add a new HTTP Listener Config global element:
+.. In General Settings, click the *+* button:
+.. Configure the following HTTP parameters, retain the default values for the other fields, and click *OK*.
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Name |HTTP_Listener_Configuration
+|Host |127.0.0.1
+|Port |8081
+|===
++
+.. Reference the HTTP Listener Configuration global element. In the General tab, specify the `/` Path.
+. Drag an Amazon S3 connector into the flow, and double-click the connector to open its properties editor.
+. If you don't have an existing Amazon S3 connector global element to choose, click the *+* sign next to *Extension Configuration* field.
+. Configure the global element properties, and then click *OK*.
+. Configure the connector parameters:
++
+image::amazon-s3-03.png[Connector Properties - General tab]
```

Comment:
This part/.adoc needs to be updated - the pictures are now obsolete and so is the code part, because the connector has slightly different XML tags.